### PR TITLE
Initial commit of task_steps_config-recipe_m2_apply_tourniquet.yaml.

### PR DESCRIPTION
**Pull request by mattbrown11**
Defines the M2 Apply Tourniquet task.

### CODE REVIEW

**Comment by mattbrown11** [comment]
@Purg @hdefazio an initial commit of the task as a starting point.

File: config/tasks/task_steps_config-recipe_m2_apply_tourniquet.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+# Reference to the file defining the mean and standard deviation of the
+# activity classifications to be used by the HMM. For N activities, both the
+# mean and standard deviation should be N x N matrices such that when activity
+# i is actually occuring, the classifier will emit confidence
+# mean[i, j] +/- std[i, j] for activity j.
+activity_mean_and_std_file: "./model_files/recipe_m2_apply_tourniquet.npy"
+
+# Task title for display purposes.
+title: "M2 Apply tourniquet"
```
Comment (Purg) [inline config/tasks/task_steps_config-recipe_m2_apply_tourniquet.yaml:16]: This will need to be just "M2" for BBN integration reasons --> this string gets set to the TaskUpdate message `task_name` field which is used to name a correlated field in the BBN Update message. To my knowledge only "M2" intersects their expected task names.

File: config/tasks/task_steps_config-recipe_m2_apply_tourniquet.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# Schema version.
+version: "1.0"
+
+# Reference to the activity classification labels configuration that we will
+# reference into.
+activity_labels: "./config/activity_labels/recipe_m2_apply_tourniquet.yaml"
```
Comment (Purg) [inline config/tasks/task_steps_config-recipe_m2_apply_tourniquet.yaml:6]: This config doesn't exist yet in the repo. Is this file intended to be added to this branch?
